### PR TITLE
Add support for Twilio Message service channels

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: templates/channels/channel_claim_twilio_message_service.haml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+-block pre-form
+  .info
+    %p
+      -blocktrans
+        You can add your Twilio Message SID as a channel. You can get your message Service SID
+        or create a new message service from <a href="https://www.twilio.com/user/account/messaging/services">Twilio Console</a>.
+
+    %p
+      -blocktrans with brand.name as brand
+        After adding your Twilio Message Service SID we will provide you with URLs endpoint for Twilio to call
```
Comment (nicpottier) [inline templates/channels/channel_claim_twilio_message_service.haml]: "with the URL endpoints"

File: templates/channels/channel_configuration.haml [hunk 2]
```diff
@@ -232,6 +232,25 @@
     %code
       https://{{ domain }}{% url 'handlers.blackmyna_handler' 'status' object.uuid %}
 
+  -elif object.channel_type == 'TMS'
+    %h4
+      -blocktrans
+        To finish configuring your Twilio Message Service connection you'll need to add the following URL your Message Service Inbound Settings.
```
Comment (nicpottier) [inline templates/channels/channel_configuration.haml]: "following URL in your Message Service Inbound Settings."

(is there a link we can provide straight to that page?)

File: templates/channels/channel_configuration.haml [hunk 3]
```diff
@@ -232,6 +232,25 @@
...
+
+    %hr
+
+    %h4
+      -trans "Request URL"
+
+    %p
+      -blocktrans
+        This endpoint should be called by Twilio when new messages are received to your Message Service.
```
Comment (nicpottier) [inline templates/channels/channel_configuration.haml]: "received by your Message Service"

**Comment by nicpottier** [comment]
Looks good. What countries do we show the Twilio message service as an option?

**Comment by norkans7** [comment]
Now I used all the countries we support on Twilio

File: templates/channels/channel_configuration.haml (hunk 2 above)
Comment (norkans7) [inline templates/channels/channel_configuration.haml]: No link since those appear on the edit page of the message service

**Comment by nicpottier** [comment]
Is that kind of messaging supported anywhere except the US? Thought it was a US only feature..

**Comment by norkans7** [comment]
Not sure about that, I send them a ticket to be sure of the list of countries supported

**Comment by nicpottier** [comment]
Looks good man!

One slight tweak in the naming and display, both on the account and channel pages, we shouldn't be repeating "Twilio Messaging Service":
  https://monosnap.com/file/DPm9Mp1nlbvNa1f48QykLVJH15NxLr

**Pull request merged**